internal/rbd/rbd_attach.go

```diff
@@ -417,8 +423,21 @@ func createPath(ctx context.Context, volOpt *rbdVolume, device string, cr *util.
 		mapArgs = append(mapArgs, "--read-only")
 	}
 
-	// Execute map
-	stdout, stderr, err := util.ExecCommand(ctx, cli, mapArgs...)
+	var (
+		stdout string
+		stderr string
+		err    error
+	)
+	netPath, nErr := util.GetNetNamespaceFilePath(util.CsiConfigFile, volOpt.ClusterID)
```

Comment:
Please set `netPath` as part of the `rbdVolume` object creation, in `NodeStageVolume` or similar.